04_Fix-Path.md

```diff
@@ -14,89 +16,52 @@ Today will we fetch of a new book with the Metafacture Playground:
 
-In this post we delve a bit deeper into ways how to point to fields in a JSON or a YAML file:
+There are top level fields like: `title` or `publish_date` which contain only text values or numbers:
 
 ```YAML
 ---
-publishers:
-- "Simon & Schuster"
-number_of_pages: "368"
-subtitle: "A Theory"
-covers:
-- "8798647"
-physical_format: "paperback"
-full_title: "Bullshit Jobs A Theory"
-key: "/books/OL27333998M"
-authors:
-- key: "/authors/OL1395062A"
-source_records:
-- "amazon:1501143336"
-- "bwb:9781501143335"
-- "marc:marc_columbia/Columbia-extract-20221130-034.mrc:71583959:3725"
-- "promise:bwb_daily_pallets_2023-05-10:W8-BRV-242"
 title: "Bullshit Jobs"
-notes: "Source title: Bullshit Jobs: A Theory"
-identifiers:
-  amazon:
-  - "1501143336"
-publish_date: "May 07, 2019"
-works:
-- key: "/works/OL20153626W"
-type:
-  key: "/type/edition"
-local_id:
-- "urn:bwbsku:W8-BRV-242"
-isbn_10:
-- "1501143336"
-isbn_13:
-- "9781501143335"
-lccn:
-- "2021276048"
-oclc_numbers:
-- "1056738022"
-classifications: {}
-lc_classifications:
-- "HF5549.5.J63 G73 2019"
-languages:
-- key: "/languages/eng"
-latest_revision: "6"
-revision: "6"
+```
+
+There are also fields like `created` that contain a nested structure where the value is again a key:value pair:
+
+```YAML
+---
 created:
   type: "/type/datetime"
   value: "2019-10-04T04:03:07.194846"
-last_modified:
-  type: "/type/datetime"
-  value: "2023-08-05T12:37:41.711036"
 ```
 
-`type.key` is called a *Path* that is JSON Path-like and points to a part of the data set - here our Yaml record - you are interested in. The data, as shown above, is structured like a tree. 
-
-There are top level simple fields like: `title`, `publish_date`, `notes` and `latest_revision` which contain only text values or numbers. Depending on the context simple fields can also be named: elemente, properties, attribute or key.
+And fields like `source_records[]` do have a list as value:
 
-There are also fields like `created` that contain a deeper structure like `type` and `value`.Nested elements that contain one or more subfields or subelements are also called objects or hash.
 
-And there are lists like `source_records[]` to which I come back later.
+```YAML
+---
+source_records:
+- "amazon:1501143336"
+- "bwb:9781501143335"
+- "marc:marc_columbia/Columbia-extract-20221130-034.mrc:71583959:3725"
+- "promise:bwb_daily_pallets_2023-05-10:W8-BRV-242"
+```
 
-Metafacture Fix is using Fix Path, a path-syntax that is JSON Path like but not identical. It also uses the dot notation but there are some differences with the path structure of arrays and repeated fields. Especially when working with JSON or YAML.
+Metafacture Fix is using FixPath to access and selectively extract data out of YAML and JSON documents. It is JSONPath like but not identical. It also uses the dot notation but there are some differences with the path structure of arrays and repeated fields.
```

Comment:
```suggestion
Metafacture Fix is using FixPath to access and selectively extract data out of (semi) structured documents. It is JSONPath like but not identical. It also uses the dot notation but there are some differences with the path structure of arrays and repeated fields.
```

No specific to JSON or Yaml, but basically works on all formats like this except when the internal parsing is special like for MARC.